Gemfile

```diff
@@ -0,0 +1,55 @@
+source 'https://rubygems.org'
```

Comment:
em đổi toàn bộ nháy đơn thành nháy đôi trong file này nhé, và xóa hết các dòng comment giúp anh